http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreterSpec.scala

```diff
@@ -155,10 +155,11 @@ class GKEInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       namespaceName = NamespaceName("ns"),
       disk = savedDisk1,
       ksaName = ServiceAccountName("app1-cromwell-ksa"),
+      gsa = WorkbenchEmail("pet123-abc@terra-test-abc.iam.gserviceaccount.com"),
       customEnvironmentVariables = envVariables
     )
 
-    res.mkString(",") shouldBe """nodeSelector.cloud\.google\.com/gke-nodepool=pool1,persistence.size=250G,persistence.gcePersistentDisk=disk1,env.swaggerBasePath=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service,config.gcsProject=dsp-leo-test1,config.gcsBucket=gs://test-bucket/cromwell-execution,config.serviceAccount.name=app1-cromwell-ksa,ingress.enabled=true,ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-from=https://1455694897.jupyter.firecloud.org,ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-to=https://leo,ingress.annotations.nginx\.ingress\.kubernetes\.io/rewrite-target=/$2,ingress.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret=ns/ca-secret,ingress.path=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service,ingress.hosts[0].host=1455694897.jupyter.firecloud.org,ingress.hosts[0].paths[0]=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service(/|$)(.*),ingress.tls[0].secretName=tls-secret,ingress.tls[0].hosts[0]=1455694897.jupyter.firecloud.org,db.password=replace-me"""
+    res.mkString(",") shouldBe """nodeSelector.cloud\.google\.com/gke-nodepool=pool1,persistence.size=250G,persistence.gcePersistentDisk=disk1,env.swaggerBasePath=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service,config.gcsProject=dsp-leo-test1,config.gcsBucket=gs://test-bucket/cromwell-execution,config.serviceAccount.name=app1-cromwell-ksa,config.serviceAccount.annotations.gcpServiceAccount=pet123-abc@terra-test-abc.iam.gserviceaccount.com,ingress.enabled=true,ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-from=https://1455694897.jupyter.firecloud.org,ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-to=https://leo,ingress.annotations.nginx\.ingress\.kubernetes\.io/rewrite-target=/$2,ingress.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret=ns/ca-secret,ingress.path=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service,ingress.hosts[0].host=1455694897.jupyter.firecloud.org,ingress.hosts[0].paths[0]=/proxy/google/v1/apps/dsp-leo-test1/app1/cromwell-service(/|$)(.*),ingress.tls[0].secretName=tls-secret,ingress.tls[0].hosts[0]=1455694897.jupyter.firecloud.org,db.password=replace-me"""
```

Comment:
TOL this is a little hard to read (and thus easy to make mistakes)... I wonder whether splitting across many lines would help, eg
```
shouldBe """nodeSelector.cloud\.google\.com/gke-nodepool=pool1,""" + 
"""persistence.size=250G,""" +
"""persistence.gcePersistentDisk=disk1,""" + 
...
```

Reply:
I +1 the idea of a multiline comment for readability. But, I'm also a biased newbie.

Reply:
I have updated it to be multi-line 👍